web/src/user_card_popover.js

```diff
@@ -883,6 +883,16 @@ function register_click_handlers() {
     }).on("success", (e) => {
         show_copied_confirmation(e.trigger);
     });
+    new ClipboardJS(".copy_link_to_profile", {
+        text(trigger) {
+            const user_id = $(trigger).attr("data-user-id");
+            const userProfileLink = window.location.origin + "/#user/" + user_id;
+
+            return userProfileLink;
```

Comment:
Let's use snake_case here.

Reply:
I have replaced the camelCase with snake_case as suggested.